### PR TITLE
updating skill description in available bidders list

### PR DESCRIPTION
**Pull request by SidelineCory24**

### CODE REVIEW

File: src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx [hunk 1]
```diff
@@ -112,7 +112,8 @@ const AvailableBidderRow = (props) => {
   const sections = isCDO ? {
     name: (<Link to={`/profile/public/${id}`}>{name}</Link>),
     status: getStatus(),
-    skill: get(bidder, 'skills[0].description') || NO_USER_SKILL_CODE,
+    // skill: <SkillCodeList skillCodes={get(bidder, 'skills')} />,
+    skill: `${get(bidder, 'skills[0].description')} (${get(bidder, 'skills[0].code')})`,
```
Comment (SidelineCory24) [inline src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx:116]: Will need to verify if we should show all skills for the small set of people who have 2 skills.

Comment (SidelineCory24) [reply src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx]: Based off of conversations, two skills should show if they have them, this causes a styling issue, but we want to keep the table flexible.

File: src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx [hunk 2]
```diff
@@ -112,7 +112,7 @@ const AvailableBidderRow = (props) => {
   const sections = isCDO ? {
     name: (<Link to={`/profile/public/${id}`}>{name}</Link>),
     status: getStatus(),
-    skill: get(bidder, 'skills[0].description') || NO_USER_SKILL_CODE,
+    skill: <SkillCodeList skillCodes={get(bidder, 'skills')} />,
```
Comment (SidelineCory24) [inline src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx:115]: NO_USER_SKILL_CODE is used in the SkillCodeList

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🥮 🍬 👍

**Comment by elizabeth-jimenez** [comment]
Good job on using existing functionality 🦜

**Pull request merged**